cg/cli/demultiplex/finish.py

```diff
@@ -21,80 +18,27 @@ def finish_group():
 
 
 @finish_group.command(name="all")
-@OPTION_BCL_CONVERTER
 @DRY_RUN
 @click.pass_obj
-def finish_all_cmd(context: CGConfig, bcl_converter: str, dry_run: bool) -> None:
+def finish_all_cmd(context: CGConfig, dry_run: bool) -> None:
     """Command to post-process all demultiplexed flow cells."""
-    demux_post_processing_api: DemuxPostProcessingNovaseqAPI = DemuxPostProcessingNovaseqAPI(
-        config=context
-    )
+    demux_post_processing_api: DemuxPostProcessingAPI = DemuxPostProcessingAPI(config=context)
     demux_post_processing_api.set_dry_run(dry_run=dry_run)
-    demux_post_processing_api.finish_all_flow_cells(bcl_converter=bcl_converter)
-
-    # Temporary finish flow cell logic will replace logic above when validated
-    demux_post_processing_api_temp: DemuxPostProcessingAPI = DemuxPostProcessingAPI(config=context)
-    demux_post_processing_api_temp.set_dry_run(dry_run=dry_run)
-    is_error_raised: bool = demux_post_processing_api_temp.finish_all_flow_cells_temp()
+    is_error_raised: bool = demux_post_processing_api.finish_all_flow_cells()
     if is_error_raised:
         raise click.Abort
 
 
 @finish_group.command(name="flow-cell")
 @click.argument("flow-cell-name")
-@OPTION_BCL_CONVERTER
 @DRY_RUN
 @click.option("--force", is_flag=True)
 @click.pass_obj
-def finish_flow_cell(
```

Comment:
we do not currently have a --force flag for the `finish_flow_cell_temporary` I will make a PR for the implementation of it

Reply:
https://github.com/Clinical-Genomics/cg/pull/2288